### PR TITLE
Pull request to merge Weather app feature branch to development branch of Microverse's weather-app project

### PR DESCRIPTION
**Pull request by henatan99**
This pull request is to merge the Weather app feature branch to the development branch of Microverse's weather-app project. This project is building a weather app to search for temperature at any place in the world, using weather API and make handy use of promises or preferably async/await functions. 
- Setup project with linters and webbpack configurations 
- Adds an async function that takes the location name as an argument and fetches weather data from open weather API. 
- Adds an await fetch function which is used to fetch the weather data from the API and the response is assigned to a variable.
- The await fetch function accepts the API URL whose path is constructed from a location name and key accessed through subscription.
- Creates descriptive reader

### CODE REVIEW

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
### Changes required :muscle: 
Hi @henatan99 :hand: 
Awesome job on this project so far.
But we need to make some changes before moving on.

- [ ] We need to have a toggle control to determine if the result will be shown in Celsius or Fahrenheit
Screenshot from 2021-03-26 12-25-43


- [ ] We need to add necessary common scripts like build, development at the package.json inside the `scripts' property. Here is a basic example (screenshot from Pathwright).
```javascript
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack serve --mode development --open" // optional, when using webpack-dev-server dependency
  },
```

This way you can remove the `mode: 'none'` from your `webpack.config.js` file.

Happy coding :computer: 
**Sérgio Torres**

File: README.md [hunk 1]
```diff
@@ -1,2 +1,65 @@
...
+## Built With
+
+- Javscript
+- HTML/CSS
+- Webpack 
+- npm 
+- package.json
+- openweather API 
+
+## Live Link
```
Comment (krys2fa) [inline README.md:42]: - [ ] [REQUIRED] Looks like you forget to add the URL of your hosted app. You can either add the URL (to make your repo professional) or take off the Live link section.

**Review by krys2fa — CHANGES_REQUESTED** [review]
## Awesome work @henatan99 👏 
Good job on implementing the changes suggested by the previous Code Reviewer. 👍 
Just a few more improvements to complete the project.
### Status: Changes Required ♻️ 

- [ ] [REQUIRED] Looks like your app does not have a toggle control to determine if the result will be shown in Celsius or Fahrenheit (**Hint**: you can make two different requests for the Celsius data and the Fahrenheit data OR make only one request and use math to convert it to the other unit) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/search_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/weather-app))



Please request another review after implementing the changes.

Happy Coding 🖥️  😃 

###### You can contact me on slack (Chris Amanor) for any clarification.

File: README.md (hunk 1 above)
Comment (henatan99) [reply README.md:42]: Hi @krys2fa, thank you for your helpful comments. For the temperature toggle, the app is working as intended. You may have a look at the gif I have attached while I was toggling the temperature between Celsius and Fahrenheit scales, in the browser app. You can also check it by clicking on the temperature. 
For the live link, I will work on it 

Regards,

chrome-capture

**Review by uche-inyama — APPROVED** [review]
## Changes Required

### Hi @henatan99 :wave: 

I was not able to interact with the application. I will leave a **gif** below, kindly use it as a reference. Consequently, I was not able to review the application based on the comment from the last reviewer.


Peek 2021-03-27 20-23

#### That's it rectify this issue, then submit another form for a code review.

### Have a great day :slightly_smiling_face: 

#### ``` Uchechukwu Henry Inyama ```

**Review by uche-inyama — APPROVED** [review]
## Approved

### Hi @henatan99 :wave: 

Great, I was able to interact with the application.

Move on with the next task.

[**Nice to have**] It is not obvious how to interact with your application. Add some information on the UI for a user to know it is a weather app. Also toggling between the temperature units, if I would suggest; there should be two buttons, which gives the user an idea that there are options to select from.

### Have a great day :slightly_smiling_face: 

#### ``` Inyama, Uchechukwu Henry ```

**Pull request merged**